convert-to-lowercase/README.md

```diff
@@ -0,0 +1,41 @@
+# Convert Files and References to Lowercase
+
+This shell script converts all filenames in a folder (and subfolders) to lowercase. Additionally, any references to the file will be replaced.
+
+Based on example from:
+http://stackoverflow.com/questions/17180580/how-to-create-a-bash-script-that-will-lower-case-all-files-in-the-current-folder
+
+## Requirements
+
+MacOSX (Windows users will need an application to run Shell scripts).
+
+## Installation
+
+Download the script and place it in your `/usr/local/bin` folder. If the folder does not exist or you have manually changed your PATH environment variable, use `echo $PATH` to print the location.
+
+Run `chmod 755 /usr/local/bin/convertToLowercase.sh` to allow the script to be run without needing superadmin privelages.
+
+Next, we'll set the locale in your bash profile so the string functions work as expected.
+
+1. Open the terminal and rum `vim ~/.bash_profile` to open your profile.
```

Comment:
Any idea what happens if the def already exists and is then appended? Yes a bootstrap script would be nice :)